### DISCUSSION TITLE
What happens if I use useLazyAsyncData with { lazy: false }?

### DISCUSSION THREAD

**Discussion by NeonMika** [nuxt/nuxt#18089] [Questions] [answered]
As the question states: **What happens if I use useLazyAsyncData with { lazy: false }**?

I do not understand why there are two differently named functions that, at least if I understood the (too complicated, too undetailed, too beginner-unfriendly) documentation right, do _exactly_ the same but _only differ in a single option_ having a different default value.

What is the reason for having two differently (in my opinion not well-)named functions for exactly the same functionallity? If I want to execute things lazy, I pass the `{ lazy: true }` option to `useAsyncData`.

Another thing: Even though we have to use `await `on `useAsyncData`, I think `useData `(without Async, and without Lazy) would be much more concise and readable. (By the way, why is `useFetch` not called `useAsyncFetch`?)

**Comment by NeonMika** [comment]
I would have really loved to receive an answer to this, probably someone can pick up on this question? :)

**Comment by danielroe** [comment, marked as answer]
It has no effect.

https://github.com/nuxt/framework/blob/00e60436017f4d2bc00a7e3445a348719c03ceff/packages/nuxt3/src/app/composables/asyncData.ts#L146-L156

**Reply by NeonMika** [reply]
Thank you @danielroe!
This should be mentioned in the data fetching docs - they are (at many places) too imprecise at the moment.

**Reply by danielroe** [reply]
PRs are really welcome 😊
